### PR TITLE
Copy changes to registration forms

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-501

Changes to header's text in the registration process forms.

### CODE REVIEW

File: config/locales/forms/registration_number_forms/en.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ en:
       new:
         title: Companies House registration number
         heading:
-          limitedCompany: What's the registration number of the company?
+          limitedCompany: What’s the company registration number?
           limitedLiabilityPartnership: What's the registration number of the limited liability partnership?
```
Comment (irisfaraway) [inline config/locales/forms/registration_number_forms/en.yml]: Should we be changing the format of the LLP text to match?

Comment (cintamani) [reply config/locales/forms/registration_number_forms/en.yml]: I am really not sure what you are asking. What's LLP?

Comment (irisfaraway) [reply config/locales/forms/registration_number_forms/en.yml]: Limited Liability Partnership. For example changing it to, "What's the limited liability partnership registration number?"

Comment (cintamani) [reply config/locales/forms/registration_number_forms/en.yml]: I'll ask Rose :)

Comment (cintamani) [reply config/locales/forms/registration_number_forms/en.yml]: I'll wait for Rose to update the ticket and open a different PR for that change. Merging this now :)

**Pull request merged**